libs/application/templates/inheritance-report/src/lib/dataSchema.ts

```diff
@@ -2,4 +2,161 @@ import * as z from 'zod'
 
 export const inheritanceReportSchema = z.object({
   approveExternalData: z.boolean().refine((v) => v),
+
+  /* assets */
+  realEstate: z.object({
+    data: z
+      .object({
+        assetNumber: z.string(),
+        description: z.string(),
+        propertyValuation: z.string().refine((v) => v),
```

Comment:
Just a thought.

Is this, and other instances of `z.string().refine((v) => v)` a `non-empty` assertion?
I don't know what is a good practice here but maybe `z.string().min(1)` is more clear if that's the case?
Does that maybe create weird error messages in the validation on the frontend?

Reply:
thought of that as well.. i dont really know which one is better

Reply:
ill do the research :)